lib/cc/pull_requests.rb

```diff
@@ -2,7 +2,28 @@ class CC::PullRequests < CC::Service
   def receive_test
     setup_http
 
-    receive_test_status
+    response = receive_test_status
```

Comment:
If I'm understanding this correctly, this `receive_test` method is called when the user uses the "test integration" button from CC, and it's meant to verify that the provided oauth token is going to be sufficient for the service to do everything it's supposed to be able to do. So previously, it checked that it was able to update statuses, and now it might additionally check if it will be able to post comments.